apps/ideaspace/src/components/modules/WorkshoppingPage/IdeaOverview/TargetAudienceCard/StyledComponents.js

```diff
@@ -0,0 +1,27 @@
+import { Card } from '../../../../common/Card/Card'
+import styled from '@emotion/styled';
+
+export const StyledCard = styled(Card)`
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  justify-content: center;
+  padding: 20px;
+  padding-left: 30px;
+  padding-right: 30px;
+  margin-top: 25px;
+`;
+
+export const Cardheading = styled.div`
```

Comment:
It looks like a lot of these `Cardheading` styled components are repeated... Could we consolidate them into a single styled component?